en/assembly/mount_gps_compass.md

```diff
@@ -15,9 +15,23 @@ The orientation follows the same frame convention as when [orienting the flight
 If you're using the normal [Compass Calibration](../config/compass.md) process (with parameter [CAL_MAG_ROT_AUTO](../advanced_config/parameter_reference.md#CAL_MAG_ROT_AUTO) enabled), the orientation should be detected automatically.
 Otherwise you can directly select the appropriate value in [CAL_MAGn_ROT](../advanced_config/parameter_reference.md#CAL_MAG1_ROT) for up to three compasses.
 
-:::warning
-You must mount the compass in a supported orientation!
+If your compass can't be oriented in any of the standard orientations, you can use a [custom rotation](#custom_orientation) as documented below.
```

Comment:
FYI you'll note that this is different from last time you saw it. I made some changes here, and then liked them so much I applied them immediately inhttps://github.com/PX4/PX4-user_guide/commit/f2428a3aa82910c9bd08a35cb0effcbe802a2f92 - and then fixed the merge conflict.

This is not just the variation against current main.

Reply:
Oh gotcha!